### DISCUSSION TITLE
yaml/yml support

### DISCUSSION THREAD

**Discussion by Netail** [biomejs/biome#2298] [Ideas]
.yaml / .yml files are widely used in the frontend ecosystem (Github workflows, configs, helm, files) which would a nice addition to supported languages.

Linked issue, for progress tracking; https://github.com/biomejs/biome/issues/2365

**Comment by Conaclos** [comment]
Hi! Would you be interested in working on this?

**Reply by ematipico** [reply]
We already have another person that showed interest on Discord

**Reply by Netail** [reply]
Seems like @castarco showed interest indeed

**Reply by castarco** [reply]
Hi @Netail , ping me on Discord :) , we can discuss it.

I didn't start working on it yet, so I don't want to stop anyone from picking it up. But I'd like to coordinate so we don't stall this topic by acident.

**Reply by Netail** [reply]
I have never worked with Rust before, so it's all yours :)

**Reply by afonsojramos** [reply]
Wouldn't mind trying to help here as well! Not a Rust pro, but I can dabble

**Comment by EpicWink** [comment]
I would like to see the configuration file (`biome.json`) support (a safe subset of[^1]) YAML syntax. YAML is a much more friendly format for writing configuration.

It seems the YAML parser is already implemented, so a good test for the parser would be to read this configuration.

[^1]: basically, only literals with JSON-equivalents (eg no date-times), and no custom types (ie no `!foo` directives). Optionally support references (ie `qux: &foo bar`, `baz: *foo`)
